build-config/conf/kernel/3.2.69/linux.x86_64.config

```diff
@@ -1507,7 +1510,7 @@ CONFIG_BCMA_POSSIBLE=y
 # CONFIG_EZX_PCAP is not set
 # CONFIG_MFD_CS5535 is not set
 # CONFIG_MFD_TIMBERDALE is not set
-# CONFIG_LPC_SCH is not set
+CONFIG_LPC_SCH=y
 # CONFIG_MFD_RDC321X is not set
```

Comment:
These configs come from machine/dell/dell_s6000_s1220/kernel/platform-dell-s6000-s1220.patch 
```
CONFIG_I2C_ISCH
CONFIG_I2C_ISMT
CONFIG_GPIO_SCH
CONFIG_MFD_CORE
CONFIG_LPC_SCH
```

As such they should *not* be set in the generic x86 config.  They should remain commented out with the `# CONFIG_XXX is not set` text.